src/puppetlabs/puppetdb/scf/storage.clj

```diff
@@ -1159,6 +1159,14 @@
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 ;;; Database support/deprecation
 
+(defn db-deprecated?
```

Comment:
This name implies it's a predicate function, which it isn't. Totally fine with the nil punning though.